src/Components/Admin/Scenes/scenecard.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { useHistory } from "react-router-dom";
+import defaultImage from "./defaultImage.svg";
```

Comment:
Extremely minor nit: can we separate library imports and imports from our own files? (just have an empty space or smth). Up to you though :)